people/templatetags/people_tags.py

```diff
@@ -14,3 +14,9 @@ def contributor_articles(contributor_page, article_page_id=None, number=None):
     if number:
         articles = articles[:number]
     return articles
+
+
+@register.filter()
```

Comment:
Shouldn't need to copy from article_tags.py, just load article_tags in the template